addons/base_automation/models/ir_actions_server.py

```diff
@@ -3,20 +3,26 @@
 from odoo.tools.json import scriptsafe as json_scriptsafe
 
 from odoo import api, exceptions, fields, models, _
+from odoo.fields import Domain
 
 from .base_automation import get_webhook_request_payload
 
 
 class IrActionsServer(models.Model):
     _inherit = "ir.actions.server"
 
-    name = fields.Char(compute='_compute_name', store=True, readonly=False)
-
     usage = fields.Selection(selection_add=[
         ('base_automation', 'Automation Rule')
     ], ondelete={'base_automation': 'cascade'})
     base_automation_id = fields.Many2one('base.automation', string='Automation Rule', ondelete='cascade')
 
+    def _get_children_domain(self):
+        # As automation rules' actions does not have a parent,
+        # we make sure multi actions can not link to automation rules' actions.
+        return super()._get_children_domain() & Domain([
+            ("base_automation_id", "=", False)
+        ])
```

Comment:
FYI,. you can call it like this
```suggestion
        return super()._get_children_domain() & Domain("base_automation_id", "=", False)
```